modal-go/examples/sandbox-prewarm/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"context"
+	"io"
+	"log"
+
+	"github.com/modal-labs/libmodal/modal-go"
+)
+
```

Comment:
Not sure we've done it for other examples, but might make sense to add a brief intro comment here explaining why and when you'd want to prewarm Images/Sandboxes?